src/patternfly/components/Alert/alert.hbs

```diff
@@ -1,3 +1,7 @@
-<div class="pf-c-alert {{ pf-c-alert--Modifier }}" aria-label="{{ pf-c-alert--AriaLabel }}">
-  {{> @partial-block }}
+<div class="pf-c-alert {{#if pf-c-alert--modifier}} {{pf-c-alert--modifier}}{{/if}}"
```

Comment:
can you drop the `pf-c` from both the modifier and the attribute here. You'll have to update the other files that call that in as well.

Reply:
done